source/Util/pre-commits.ts

```diff
@@ -0,0 +1,14 @@
+import { IntegrationType } from '../IIntegrationConfig';
```

Comment:
Totally nitpicky, but I'd use the the singular form, since the code operates on a single commit (`pre-commit.ts`). FWIW, git itself does hyphenate this term, so that seems proper. :-).

`pre-push` might have been one other option worth looking at if we feel like it is disruptive at pre-commit time. It's not as thorough as `pre-commit`, though.